src/ds/d500/d500-fw-update-device.cpp

```diff
@@ -98,4 +108,49 @@ ds_d500_update_device::ds_d500_update_device( std::shared_ptr< const device_info
 
         return true;
     }
+
+    void ds_d500_update_device::dfu_manifest_phase(const platform::rs_usb_messenger& messenger, rs2_update_progress_callback_sptr update_progress_callback) const
+    {
+        // measuring the progress of the writing to flash (when enabled by FW)
+        if (!wait_for_manifest_completion(messenger, RS2_DFU_STATE_DFU_MANIFEST, std::chrono::seconds(200), update_progress_callback))
```

Comment:
Should we lower the timeout?
This will also be the timeout for D400 devices that only should take several seconds?

Reply:
The timeout is around 1.5 times the time it takes.
It is not related to the D400 devices, this code is only for D500 devices.